crates/red_knot_python_semantic/src/types.rs

```diff
@@ -75,13 +75,44 @@ pub(crate) fn global_symbol_ty_by_name<'db>(db: &'db dyn Db, file: File, name: &
     symbol_ty_by_name(db, global_scope(db, file), name)
 }
 
-/// Shorthand for `symbol_ty` that looks up a symbol in the builtins.
+/// Enumeration of various core stdlib modules, for which we have dedicated Salsa queries.
 ///
-/// Returns `Unbound` if the builtins module isn't available for some reason.
+/// Things will start getting very strange during type-checking if one of these doesn't exist.
```

Comment:
I would remove this comment. It's unclear to a reader what they should do about it.